### DISCUSSION TITLE
How to follow releases?

### DISCUSSION THREAD

**Discussion by vasiliy-t** [actions/virtual-environments#4362] [Ideas]
Hi there! Thank you for great job done! This saves us a lot of time and effort!

We are using fork of virtual-environments adapted to build openstack images. Now we are updating manually: set virtual-environments to specific commit then patch it with openstack specific build instructions then build. We want to setup automatic upgrade process. 
Please provide some guidance on how to follow virtual-environment releases: I see releases prefixed with win/ mac etc - are those releases are specific to this OSes? We need only Ubuntu 20 for now. Also there are a bunch of release- branches - should we upgrade from those branches or follow master?

**Comment by miketimofeev** [comment]
Hey! The releases are mostly needed to update the documentation for hosted VMs. If you build your own VMs you can just follow the main branch as it's the most actual one and we merge all the fixes into it.

**Reply by vasiliy-t** [reply]
Looks good for me, will follow main branch.
